lib/internal/event_target.js

```diff
@@ -59,6 +59,7 @@ const kStop = Symbol('kStop');
 const kTarget = Symbol('kTarget');
 const kHandlers = Symbol('kHandlers');
 const kWeakHandler = Symbol('kWeak');
+const kForcedHandler = Symbol('kForcedHandler');
```

Comment:
```suggestion
const kResistStopPropogation = Symbol('kResistStopPropogation');
```
"force" is a very generic name, and I think we can give this symbol a name that explains the use case

Reply:
@benjamingr offered `kAlwaysRun`, but I find `kResistStopPropogation` more descriptive. I'll go with `kResistStopPropogation `, unless anyone has a strong opinion against this 🙂